### PR TITLE
#794 - Added bike filters.

### PR DESCRIPTION
**Pull request by aviklai**

### CODE REVIEW

File: anyway/models.py [hunk 1]
```diff
@@ -176,6 +176,7 @@ class AccidentMarker(MarkerMixin, Base):
     cross_location = Column(Integer)
     cross_direction = Column(Integer)
     involved = relationship("Involved", foreign_keys="Involved.accident_id")
+    vehicle = relationship("Vehicle", foreign_keys="Vehicle.accident_id")
```
Comment (danielhers) [inline anyway/models.py]: Shouldn't this be plural `vehicles`, since it's a many-to-many relationship?

File: anyway/models.py [hunk 2]
```diff
@@ -331,7 +332,17 @@ def bounding_box_query(is_thin=False, yield_per=None, involved_and_vehicles=Fals
         if kwargs.get('surface', 0) != 0:
             markers = markers.filter(AccidentMarker.road_surface == kwargs['surface'])
         if kwargs.get('acctype', 0) != 0:
-            markers = markers.filter(AccidentMarker.subtype == kwargs['acctype'])
+            if kwargs['acctype'] <= 20:
+                markers = markers.filter(AccidentMarker.subtype == kwargs['acctype'])
+            elif kwargs['acctype'] == 21:
+                markers = markers.\
+                    filter(AccidentMarker.vehicle.any(Vehicle.vehicle_type == 15)).group_by(AccidentMarker.id).\
```
Comment (danielhers) [inline anyway/models.py]: I think these numeric constants should definitely be defined somewhere, at least as Python constants. This is true regarding the other constants in thsi function, but at least let's do it for new code.

File: templates/index.html [hunk 3]
```diff
@@ -853,6 +853,10 @@ <h4 class="modal-title">העדפות:</h4>
                 <label for="checkbox-type-pedestrian" class="btn one">אחר</label>
                 <input type="radio" name="acctype" id="checkbox-type-else" value=1 onclick="app.loadFilter()" />
                 <label for="checkbox-type-else" class="btn one">פגיעה בהולך רגל</label>
+                <input type="radio" name="acctype" id="checkbox-type-bike-no-cas" value=21 onclick="app.loadFilter()" />
+                <label for="checkbox-type-bike-no-cas" class="btn one">אופניים ללא נפגעים</label>
+                <input type="radio" name="acctype" id="checkbox-type-with-cas" value=22 onclick="app.loadFilter()" />
+                <label for="checkbox-type-with-cas" class="btn one">אופניים עם נפגעים</label>
```
Comment (danielhers) [inline templates/index.html]: You forgot the `bike` in the second radio button. Also, any reason not to write the whole word `casualty`? Or maybe `injured` would be more accurate ("casualty" entails death, at least in my mind).

**Comment by simonaho** [comment]
תודה אבי.

**Comment by aviklai** [comment]
@danielhers Done.

**Pull request merged**